### ISSUE TITLE
Improve IDL handling in `map_cruz()`

### ISSUE THREAD

**Issue by ericmkeen** [PIFSC-Protected-Species-Division/LTabundR#5]
Currently polygons that cross the international date line (IDL) are displayed inconveniently in the `leaflet`-based mapping function `map_cruz()`. Work on improving!

**Comment by ericmkeen** [comment]
fyi same issue exists for the function `strata_explore()`

**Comment by ericmkeen** [comment]
I think the best thing to do is here is wait for this issue to be resolved by `leaflet()`. Clunky longitude wrapping is a [known issue](https://github.com/Leaflet/Leaflet/issues/2333) with lots of folks wanting an update; I think it would be best to get the solution from them rather than us code us something ourselves.

**Comment by ericmkeen** [comment]
To reduce inconvenience for use of `map_cruz()`, I have added a feature that centers the initial view of the leaflet map based on the average longitude of the DAS coordinates; I have also added an input that lets you set the initial zoom level, with a default of 4. This makes the initial view of the interactive map less susceptible to disruptive effects of the date line. This change will be live in version 1.3.

I wlil close this for now and we can re-open if there is no movement from the leaflet team.
